sentry_youtrack/forms.py

```diff
@@ -249,6 +258,10 @@ def get_youtrack_client(self, data, additional_params=None):
             self.client_errors['url'] = self.error_message['invalid_ssl']
         if client:
             try:
```

Comment:
@bogdal By doing this, I can validate the URL but it happened at the second step.
I mean after enter:
- a wrong url
- username
- password

and press Save Changes, it switched to the second form to let us choose Linked project, Default tags, Ignore fields. The validation happened at this form with 2 errors:
1. Cannot get any project from this URL.
2. Invalid project: 'None'

The URL is saved into the database and there is no way to change it (except login to the db).
Linked project is empty.

So, my question is: how can I validate the URL before saving at the first step?